chain/types/actor.go

```diff
@@ -26,7 +26,7 @@ type ActorV5 struct {
 	Head    cid.Cid
 	Nonce   uint64
 	Balance BigInt
-	// Predictable Address
+	// Deterministic Address: f1, f3, or f4 address.
```

Comment:
This can only be an f4 address (...for now)?

Reply:
Correct. We had discussed an alternative version of the world where we hoisted key addresses here, but we didn't go through with that.